rabbitmq/tests/metrics.py

```diff
@@ -0,0 +1,51 @@
+# (C) Datadog, Inc. 2018
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+COMMON_METRICS = [
+    'rabbitmq.node.fd_used',
+    'rabbitmq.node.disk_free',
+    'rabbitmq.node.mem_used',
+    'rabbitmq.node.run_queue',
+    'rabbitmq.node.sockets_used',
+    'rabbitmq.node.partitions',
+    'rabbitmq.node.running',
+    'rabbitmq.node.disk_alarm',
+    'rabbitmq.node.mem_alarm',
+]
+
+E_METRICS = [
+    'messages.confirm.count',
+    'messages.confirm.rate',
+    'messages.publish_in.count',
+    'messages.publish_in.rate',
+    'messages.publish_out.count',
+    'messages.publish_out.rate',
+    'messages.return_unroutable.count',
+    'messages.return_unroutable.rate',
+    # TODO: create a 'fake consumer' and get missing metrics
```

Comment:
I am a big `TODO` hater in the source code. Can we do something about this?

Reply:
I didn't put this there, it was already there. Tbh I don't really know how I would go about creating a fake consumer.

Reply:
I know you didn't add it. I was thinking to either fix it, make it as a note or just remove it, don't you think?